conf/eslint.json

```diff
@@ -10,11 +10,11 @@
     },
 
     "rules": {
-        "no-alert": 2,
-        "no-array-constructor": 2,
+        "no-alert": 0,
```

Comment:
Those should all still stay enabled. config.js should be modified to not use them unless "extends:eslint-recommended" is specified.